src/darwin/Framework/CHIP/MTRDeviceControllerFactory.h

```diff
@@ -66,44 +66,47 @@ NS_ASSUME_NONNULL_BEGIN
  * Whether to run a server capable of accepting incoming CASE
  * connections.  Defaults to NO.
  */
-@property (nonatomic, assign) BOOL startServer;
+@property (nonatomic, assign) BOOL shouldStartServer;
 
 - (instancetype)init NS_UNAVAILABLE;
 - (instancetype)initWithStorage:(id<MTRPersistentStorageDelegate>)storageDelegate;
 @end
 
-@interface MTRControllerFactory : NSObject
+@interface MTRDeviceControllerFactory : NSObject
 
-@property (readonly, nonatomic) BOOL isRunning;
-
-- (instancetype)init NS_UNAVAILABLE;
-+ (instancetype)new NS_UNAVAILABLE;
```

Comment:
I thought that making `init` and `new` unavailable was to make sure folks are using the shared instance ?  Do we really want to not lock it down ?

Reply:
I just moved those to the end of the interface.